vcpkg/contributing/maintainer-guide.md

```diff
@@ -12,6 +12,29 @@ It is intended to serve the role of
 [Homebrew's Maintainer Guidelines](https://docs.brew.sh/Maintainer-Guidelines), and
 [Homebrew's Formula Cookbook](https://docs.brew.sh/Formula-Cookbook).
 
+## Overall Registry Design Goals
+
+### Ports in the current baseline must be simultaneiously installable
+
+We wish to be able to show downstream customers of libraries in the default registry that the

```

Comment:
```suggestion
We wish to be able to show downstream users of libraries in the curated registry that the
```

Reply:
Billy: (Or at least, that is the intent and uses of 'default' to mean github.com/microsoft/vcpkg are bugs)

Also Billy: (this PR)

:sigh: